WcaOnRails/vendor/assets/javascripts/bootstrap-datetimepicker.js

```diff
@@ -1019,7 +1019,7 @@
                 if (component && component.hasClass('btn')) {
                     component.toggleClass('active');
                 }
-                widget.show();
+                widget.css("display", "block");
```

Comment:
I don't get it.
I investigated why the datetimepicker wouldn't show up, and upon digging a bit I found out it was correctly added to the dom, but the style had a "display:none" even after the show...
With that fix it works alright, but it's really puzzling...